--- Resources/app/administration/src/module/sw-media/component/sidebar/sw-media-quickinfo/index.js       2023-07-24 14:28:36
+++ Resources/app/administration/src/module/sw-media/component/sidebar/sw-media-quickinfo/index.js   2023-04-17 02:14:27
@@ -109,7 +109,7 @@

         copyLinkToClipboard() {
             if (this.item) {
-                dom.copyToClipboard(this.item.url);
+                dom.copyToClipboard(this.item.url.replace('https://rafu-zentaurus.de/', 'https://rafu.de/'));
                 this.createNotificationSuccess({
                     message: this.$tc('sw-media.general.notification.urlCopied.message'),
                 });
