--- Resources/app/administration/src/core/shopware.ts       2023-07-24 14:28:36
+++ Resources/app/administration/src/core/shopware.ts   2023-04-17 02:14:27
@@ -240,8 +240,8 @@
     };

     public Defaults = {
-        systemLanguageId: '2fbb5fe2e29a4d70aa5854ce7ce3e20b',
-        defaultLanguageIds: ['2fbb5fe2e29a4d70aa5854ce7ce3e20b'],
+        systemLanguageId: '7586b51f54c5466bb3a1ef0a3e7e67c7',
+        defaultLanguageIds: ['7586b51f54c5466bb3a1ef0a3e7e67c7'],
         versionId: '0fa91ce3e96a4bc2be4bd9ce752c3425',
         storefrontSalesChannelTypeId: '8a243080f92e4c719546314b577cf82b',
         productComparisonTypeId: 'ed535e5722134ac1aa6524f73e26881b',
